test/unit/org/apache/cassandra/hints/HintsCatalogTest.java

```diff
@@ -138,14 +136,27 @@ public void deleteHintsTest() throws IOException
     @Test
     public void exciseHintFiles() throws IOException
     {
-        File directory = new File(Files.createTempDirectory(null));
-        try
-        {
-            exciseHintFiles(directory);
-        }
-        finally
+        File directory = new File(testFolder.newFolder());
+        exciseHintFiles(directory);
+    }
+
+    @Test
+    public void hintsTotalSizeTest() throws IOException
+    {
+        File directory = new File(testFolder.newFolder());
+        UUID hostId = UUID.randomUUID();
+        long now = System.currentTimeMillis();
```

Comment:
Maybe use `Clock.Global.currentTimeMillis()` here. I've seen tests being changed to use `Clock.Global.currentTimeMillis()`. See https://github.com/apache/cassandra/commit/599294c919df10a4aea592d78364af565d8ff1ed

Reply:
Good point!